docs/source/markdown/podman-image-exists.1.md

```diff
@@ -1,34 +1,44 @@
 % podman-image-exists(1)
 
 ## NAME
+
 podman-image-exists - Check if an image exists in local storage
 
 ## SYNOPSIS
+
 **podman image exists** *image*
```

Comment:
If you name it `*image*` here, you should also name it `*image*` in the other manpages.
For example in `podman-image-diff`.